tests/unit/openpype/modules/sync_server/test_site_operations.py

```diff
@@ -0,0 +1,129 @@
+"""Test file for Sync Server, tests site operations add_site, remove_site.
+
+    File:
+        creates temporary directory and downloads .zip file from GDrive
+        unzips .zip file
+        uses content of .zip file (MongoDB's dumps) to import to new databases
+        with use of 'monkeypatch_session' modifies required env vars
+            temporarily
+        runs battery of tests checking that site operation for Sync Server
+            module are working
+        removes temporary folder
+        removes temporary databases (?)
+"""
+import pytest
+
+from tests.lib.testing_wrapper import TestCase
+from bson.objectid import ObjectId
+
+
+class TestSiteOperation(TestCase):
+
+    @pytest.fixture(scope="module")
+    def setup_sync_server_module(self, dbcon):
+        """Get sync_server_module from ModulesManager"""
+        from openpype.modules import ModulesManager
+
+        manager = ModulesManager()
+        sync_server = manager.modules_by_name["sync_server"]
+        yield sync_server
+
+    @pytest.mark.usefixtures("dbcon")
+    def test_project_created(self, dbcon):
+        assert ['test_project'] == dbcon.database.collection_names(False)
```

Comment:
Not sure what's the current pymongo version is being used, but if `>=3.7`, better change to use [`list_collection_names`](https://pymongo.readthedocs.io/en/stable/api/pymongo/database.html#pymongo.database.Database.list_collection_names) since `collection_names` has been deprecated and may get lots of `DeprecationWarning` in test.

```python
# copied from doc : exclude system collections with `list_collection_names`
filter = {"name": {"$regex": r"^(?!system\.)"}}
db.list_collection_names(filter=filter)
```